Solaris-specific system paths. Our bash configuration lives in /etc/bash/inputrc.

--- doc/bash.info.orig	2020-11-18 21:13:06.000000000 +0000
+++ doc/bash.info	2020-12-08 12:17:05.628090856 +0000
@@ -8788,7 +8788,7 @@
 directory.  The name of this file is taken from the value of the shell
 variable ‘INPUTRC’.  If that variable is unset, the default is
 ‘~/.inputrc’.  If that file does not exist or cannot be read, Readline
-looks for ‘/etc/inputrc’.  The ‘bind’ builtin command can also be used
+looks for ‘/etc/bash/inputrc’.  The ‘bind’ builtin command can also be used
 to set Readline keybindings and variables.  *Note Bash Builtins::.
 
    When a program that uses the Readline library starts up, Readline
@@ -9424,8 +9424,8 @@
 ‘$include’
      This directive takes a single filename as an argument and reads
      commands and key bindings from that file.  For example, the
-     following directive reads from ‘/etc/inputrc’:
-          $include /etc/inputrc
+     following directive reads from ‘/etc/bash/inputrc’:
+          $include /etc/bash/inputrc
 
 
 File: bash.info,  Node: Sample Init File,  Prev: Conditional Init Constructs,  Up: Readline Init File
@@ -8155,8 +8155,8 @@
      # Lines beginning with '#' are comments.
      #
      # First, include any system-wide bindings and variable
-     # assignments from /etc/Inputrc
-     $include /etc/Inputrc
+     # assignments from /etc/bash/Inputrc
+     $include /etc/bash/Inputrc
 
      #
      # Set various bindings for emacs mode.
--- doc/bashref.info.orig	2020-11-18 21:12:25.000000000 +0000
+++ doc/bashref.info	2020-12-08 12:16:14.823974296 +0000
@@ -8789,7 +8789,7 @@
 directory.  The name of this file is taken from the value of the shell
 variable ‘INPUTRC’.  If that variable is unset, the default is
 ‘~/.inputrc’.  If that file does not exist or cannot be read, Readline
-looks for ‘/etc/inputrc’.  The ‘bind’ builtin command can also be used
+looks for ‘/etc/bash/inputrc’.  The ‘bind’ builtin command can also be used
 to set Readline keybindings and variables.  *Note Bash Builtins::.
 
    When a program that uses the Readline library starts up, Readline
@@ -9425,8 +9425,8 @@
 ‘$include’
      This directive takes a single filename as an argument and reads
      commands and key bindings from that file.  For example, the
-     following directive reads from ‘/etc/inputrc’:
-          $include /etc/inputrc
+     following directive reads from ‘/etc/bash/inputrc’:
+          $include /etc/bash/inputrc
 
 
 File: bashref.info,  Node: Sample Init File,  Prev: Conditional Init Constructs,  Up: Readline Init File
@@ -8155,8 +8155,8 @@
      # Lines beginning with '#' are comments.
      #
      # First, include any system-wide bindings and variable
-     # assignments from /etc/Inputrc
-     $include /etc/Inputrc
+     # assignments from /etc/bash/Inputrc
+     $include /etc/bash/Inputrc
 
      #
      # Set various bindings for emacs mode.
--- doc/bash.1.orig	2020-11-18 21:03:59.000000000 +0000
+++ doc/bash.1	2020-12-08 12:21:43.110560302 +0000
@@ -6632,7 +6632,7 @@
 If that variable is unset, the default is
 .FN \*~/.inputrc .
 If that file  does not exist or cannot be read, \fBreadline\fP looks for
-.FN /etc/inputrc .
+.FN /etc/bash/inputrc .
 When a program that uses the \fBreadline\fP library starts up,
 \fBreadline\fP reads the initialization file
 and sets the key bindings and variables found there,
@@ -7398,11 +7398,11 @@
 This directive takes a single filename as an argument and reads commands
 and key bindings from that file.
 For example, the following directive would read
-.FN /etc/inputrc :
+.FN /etc/bash/inputrc :
 .PP
 .RS
 .nf
-\fB$include\fP \^ \fI/etc/inputrc\fP
+\fB$include\fP \^ \fI/etc/bash/inputrc\fP
 .fi
 .RE
 .SS Searching
--- lib/readline/rlconf.h.orig	2017-02-21 16:25:30.000000000 +0000
+++ lib/readline/rlconf.h	2020-12-08 12:19:52.079293614 +0000
@@ -43,7 +43,7 @@
 #define DEFAULT_INPUTRC "~/.inputrc"
 
 /* The ultimate last-ditch filename for an init file -- system-wide. */
-#define SYS_INPUTRC "/etc/inputrc"
+#define SYS_INPUTRC "/etc/bash/inputrc"
 
 /* If defined, expand tabs to spaces. */
 #define DISPLAY_TABS
